### PR TITLE
samantha-week3

### PR DESCRIPTION
**Pull request by samiachi**

### CODE REVIEW

File: trial.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+console.log("hello world");
```
Comment (KrustyC) [inline trial.js]: Please remove this `console.log `

**Comment by samiachi** [comment]
My week three changes were accidentally added to this trial commit

File: week-3-practice/exercise-2.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+//
+//}
+var shortNameThatStartsWithS = namesOfMusicalArtists.find(name => {
+  return name.length < 5 && name[0] == "S";
+});
+
+console.log(shortNameThatStartsWithS);
+
+// Expected output = "Sade"
+//BTW, Yes Sade is the best artist from that array, it’s a close call between her and Prince, but don’t start talking to me about Adele, please.
```
Comment (martynamaron) [inline week-3-practice/exercise-2.js]: Elivs was pretty cool too y'know... 🙄

File: week-3-practice/exercise-2.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "Beatles",
+  "Madonna",
+  "Adele"
+];
+
+//function isShortWiths(name) {
+//
+//}
+var shortNameThatStartsWithS = namesOfMusicalArtists.find(name => {
+  return name.length < 5 && name[0] == "S";
```
Comment (martynamaron) [inline week-3-practice/exercise-2.js]: Well done! 👏

File: week-3-practice/exercise-3.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+// You know that Abz is shortened from his full name "Abdullah", and Bart is shortened from “Bartholomew” (Yes my parents did that to me and high school was FULL of Simpsons and biblical reference jokes before you ask) ,you want to check if the the array has any shortened names.
+
+// Write a function that checks for abbreviated names (4 characters or less)
+
+// write code here
+function isNickname(name) {
+  return name.length <= 4;
+}
+
+if (names.some(isNickname)) {
```
Comment (martynamaron) [inline week-3-practice/exercise-3.js]: Nice! Knowing that `some()` returns a boolean do you think you still need the if/else?

File: week-3-practice/exercise.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+//var numbersDividedBy2 = numbers.map(function(number) {
+//    return number / 2;
+//});
+
+var numbersDividedBy2 = numbers.map(number => {
+  return number / 2;
+});
+
+console.log(numbersDividedBy2);
```
Comment (martynamaron) [inline week-3-practice/exercise.js]: Great! I think there is one more type of syntax you could use though! :)

**Pull request closed without merging**